push_data.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+require 'functions.php';
+
+$db = new PDO('mysql:host=127.0.0.1;dbname=portfolio', 'root');
+
+$bio_push = $_POST['bio'];
+$life_now_push = $_POST['life_now'];
+$goals_push = $_POST['goals'];
+
+
+
+push_data($bio_push,$life_now_push,$goals_push,$db);
+
+?>
+
+<html>
+    <a href="form.php"><h3>click here to return to form</h3></a>
+    <a href="index.php"> <h3>click here to go back to webpage</h3></a>
+</html>
```

Comment:
Function Push_data has no doc bloc

Reply:
Function about_output has no doc bloc.

Reply:
Take the functions out of the files which use them and then require them in to add seperations of concern and so that when you run the tests it doesn't run a whole page.

Reply:
What is this file, if it isn't on use it shouldn't be here. gitignore please.

Reply:
NO TESTS WORK

Reply:
You have duplicate mages in your project. Keep them all in the img folder.

Reply:
There are no failure tests in here or type matching tests. Please update.